internal/server/spanner/dsutil.go

```diff
@@ -25,6 +25,15 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const (
+	// Indicates that all properties should be returned.
+	WILDCARD = "*"
+	// Indicates that
+	CHAIN = "+"
+	// Maximum number of edge hops to traverse for chained properties.
+	MAX_HOPS = 10
```

Comment:
nit: Is max hops used in other places or only when making spanner queries? If so, consider moving it to `query.go`.

Reply:
Done